### PR TITLE
5.x

### PR DESCRIPTION
**Pull request by micw**
I have created a branch 5.x that fixes some compatibility issues with ES 5.0.0-alpha1 and above

### CODE REVIEW

**Comment by jippi** [comment]
any love for this PR ? :)

**Comment by nezhazheng** [comment]
why not merge yet?

**Comment by bmandrade** [comment]
Hey guys, could this one be merged? I'm really interested in it

**Comment by phutchins** [comment]
Yes, lets get this merged. Going to have to remove kopf until it is and I don't like that! :)

**Comment by danielmotaleite** [comment]
Please merge this, cerebro is still incomplete, having a kopf working better with ES 5.x until it is ready will make everyone life easier

**Comment by g6n** [comment]
I have a working one, with the fixes from mitw 

docker pull g6nuk/elasticsearch-kopf:5.x 

Some other parts are still missing in the mitw fix, hence I forked yet another "version”

**Comment by micw** [comment]
Sorry, I had no time during holidays. I'll check it tomorrow or monday 
and merge.
Thank you for sharing this!

**Comment by fperks** [comment]
Any update on kopf? Moving to ES5 soon and i need atleast kopf.

**Comment by nicholaskuechler** [comment]
+1

**Comment by portante** [comment]
Please merge this!  We need ES 5 supports, and cerebro seems a long way off.

File: docker/Dockerfile [hunk 1]
```diff
@@ -14,8 +14,8 @@ ADD nginx.conf.tpl /etc/nginx/nginx.conf.tpl
 ADD ./run.sh ./run.sh
 
 # kopf
-ENV KOPF_VERSION 2.0.1
-RUN curl -s -L "https://github.com/lmenezes/elasticsearch-kopf/archive/v${KOPF_VERSION}.tar.gz" | \
+ENV KOPF_VERSION 5.0.0
+RUN curl -s -L "https://github.com/micw/elasticsearch-kopf/archive/v${KOPF_VERSION}.tar.gz" | \
```
Comment (portante) [inline docker/Dockerfile]: To merge, this needs an adjustment.

**Comment by lmenezes** [comment]
sorry for leaving this thread unattended for so long. 

My time available for open source is very limited atm and I focused on getting cerebro on par with kopf. 

It is pretty much at this stage already, although I'm still working on polishing some not so smooth edges and adding new features.

**Pull request closed without merging**